content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
++++
+title = "Community Membership"
+description = "Guidelines for contributing to Kubeflow"
+weight = 30
+aliases = ["/docs/membership/"]
++++
+
+
+**Note:** This document is a work in progress
+
+This document outlines the various responsibilities of contributor roles in Kubeflow. Kubeflow is divided into working groups that have stewardship over different subprojects/repositories
+
+Responsibilities for most roles are scoped to these repositories.
+
+| Role | Responsibilities | Requirements | Defined by |
```

Comment:
Can we also add the steering committee here?

Reply:
Is there documentation anywhere for the KSC that I can pull from?

Reply:
The only doc is available here https://github.com/kubeflow/community/blob/master/KUBEFLOW-STEERING-COMMITTEE.md

Reply:
I'm concerned about having conflicting documents.. Should I just link to the KSC document from here?

Reply:
you can provide a short description of the role under "responsibilities" and link back to KSC doc for "requirement" and "defined by"